app/datatables/batch_process_datatable.rb

```diff
@@ -14,10 +14,10 @@ def view_columns
     # Declare strings in this format: ModelName.column_name
     # or in aliased_join_table.column_name format
     @view_columns ||= {
-      process_id: { source: "BatchProcess.id" },
-      user: { source: "BatchProcess.user_id" },
-      time: { source: "BatchProcess.created_at" },
-      size: { source: "BatchProcess.oid" },
+      process_id: { source: "BatchProcess.id", cond: :like, searchable: true },
+      user: { source: "BatchProcess.user_id", cond: :like, searchable: true },
+      time: { source: "BatchProcess.created_at", cond: :like, searchable: true },
+      size: { source: "BatchProcess.oid", cond: :like, searchable: true },
```

Comment:
do you think this is part of the error?
I don't mind that they're here; we just left them off because these are the default values.

Reply:
hrm, possibly not, but I was getting an error locally that it couldn't find the search index or something, and this fixed it? And I thought we were seeing search wasn't working in production? Can't hurt might help?